### DISCUSSION TITLE
Possibility to add a drill-down functionality?

### DISCUSSION THREAD

**Discussion by katalin2003** [themustafaomar/jsvectormap#55] [Q&A]
Hello and thank you for this library!
I’m planning on using it on a small project. Mostly because its lack of dependency on jQuery.
I would like to know if a drill-down functionality is already supported or planned and how could I help in any way?
Thank you @themustafaomar

**Comment by themustafaomar** [comment]
Very sorry for the late response, don't know how I didn't notice this!

About drill-down, it's not supported currently, not sure maybe soon I'll create a plugin for it.
